wiki/Skin/en.md

```diff
@@ -4,7 +4,29 @@ stub: true
 
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned." 
```

Comment:
"Many players regularly engage in the use of skins" this just sounds beyond weird.. It is also not necessarily "disliking the default skin" but often just prefering e.g. their favourite waifu over it
```suggestion
A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players use skins to allow for a more customised user experience. Almost all of what is visible or audible in osu! can be "skinned." 
```